app/index.html

```diff
@@ -264,9 +264,14 @@
                on-node-toggle="on_toggle(node, expanded)"
                expanded-nodes="expanded_nodes">
           <!-- ArchivesSpace record -->
-          <span ng-if="node.type === 'resource' || node.type === 'resource_component'" tree-droppable on-drop="drop" file-type="record">
+          <span ng-if="node.type === 'resource'" tree-droppable on-drop="drop" file-type="record">
             {{ node.title }} <div ng-if="node.identifier">({{ node.identifier }})</div>
           </span>
+          <span ng-if="node.type === 'resource_component'" tree-droppable on-drop="drop" file-type="record">
+            <div ng-if="node.display_title">{{ node.display_title }}</div>
+            <div ng-if="!node.display_title">{{ node.title }}</div>
```

Comment:
I don't think we're styling it differently; it has a `<div>` just so I can tack on the if statement.